app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,17 @@
+require "stripe"
+
+class PaymentsController < ApplicationController
+  def charge
+    token = params['token']
+    return unless token
+
+    Stripe.api_key = ENV['STRIPE_SECRET_KEY']
```

Comment:
Ideally, I could update Heroku environment from development to production and all our configs for Stripe, Lob, etc. would "just work". Right now, there's only one value for each of these configs in Heroku and we'll have to manually update each of them to flip the switch from development to production.

I suppose this is OK for us at this stage. In practice, I'm guessing most projects have a dev site, where everything points to test credentials, and a production site, where everything points to live credentials?